docs/CMakeLists.txt

```diff
@@ -76,7 +76,7 @@ if(LIBCELLML_COVERAGE)
     COMMENT "Running coverage tests with html output")
 endif()
 
-if(SPHINX_FOUND OR DOXYGEN_FOUND OR LIBCELLML_COVERAGE)
+if(SPHINX_FOUND AND DOXYGEN_FOUND AND LIBCELLML_COVERAGE)
```

Comment:
This shouldn't be an AND logic, the docs target gets created for any of the possible documentation related commands.  The remove _directory doesn't seem to mind missing directories.

Reply:
I don't like it, but nevermind: b19bdade7.